ledger-core/v2/reference/global.go

```diff
@@ -58,54 +58,46 @@ func (v *Global) WriteTo(w io.Writer) (int64, error) {
 	return n + n2, err
 }
 
-func (v *Global) Read(p []byte) (int, error) {
-	n, err := v.addressLocal.Read(p)
-	if err != nil || n < v.addressLocal.len() {
-		return n, err
-	}
-	n2, err := v.addressBase.Read(p[n:])
-	return n + n2, err
+func (v Global) Read(b []byte) (int, error) {
```

Comment:
this function should still be *Global, if it supposed to modify Global object, same goes to Local, I guess

Reply:
nope, Read puts data into the buffer. Please refer to io.Read
But actually this is misuse of io.Read because Read expects that last position is remembered.
This method must be removed as soon as we change serialization approach.